weather_stats/weatherstatsapp/README.md

```diff
@@ -0,0 +1,31 @@
+# Weather Stats App
+
+## Generating gvmi image hash
+
+Use [gvmkit-build](https://golem-network.gitbook.io/golem-sdk-develop/requestor-tutorials/vm-runtime/convert-a-docker-image-into-a-golem-image) to generate a hash
+
+```sh
+docker compose build --no-cache
+gvmkit-build weatherstatsapp-app:latest
+gvmkit-build weatherstatsapp-app:latest --push
+```
+
+## Running local docker instance
+
+Create network
+
+```sh
+docker network create weather-stats
+```
+
+Replace [nginx](nginx.conf) proxy_pass configuration
```

Comment:
Is there any reason to have that requirement?

Reply:
as I understand, the nginx config is tailored for running on Golem (which differs from the Docker setup now) and thus, in order to be run on Docker, it needs to be tweaked back...